src/main/kotlin/edu/wpi/first/wpilib/javastyle/WPILibJavaStylePlugin.kt

```diff
@@ -4,13 +4,19 @@ import groovy.util.XmlParser
 import org.gradle.api.Plugin
 import org.gradle.api.Project
 import org.gradle.api.plugins.JavaPluginConvention
-import org.gradle.api.plugins.quality.*
+import org.gradle.api.plugins.quality.CheckstyleExtension
+import org.gradle.api.plugins.quality.CheckstylePlugin
+import org.gradle.api.plugins.quality.FindBugs
+import org.gradle.api.plugins.quality.FindBugsExtension
+import org.gradle.api.plugins.quality.FindBugsPlugin
+import org.gradle.api.plugins.quality.PmdExtension
+import org.gradle.api.plugins.quality.PmdPlugin
 
 class WPILibJavaStylePlugin : Plugin<Project> {
 
     override fun apply(project: Project) {
 
-        val extension = project.extensions.create("javastyle", WPILibJavaStylePluginExtension::class.java)
+        val extension = project.extensions.create("WPILibJavaStyle", WPILibJavaStylePluginExtension::class.java)
 
         // Only for java projects
         project.pluginManager.withPlugin("java") {
```

Comment:
All of the code in this block needs to be in an `afterEvaluate` block or it won't use the changed values the user modifies when configuring the `WPILibJavaStylePluginExtension` it will only use the defaults.

Reply:
See #2

Reply:
Sounds good!